docs/run-platform-server/configuring.mdx

```diff
@@ -0,0 +1,154 @@
+---
+title: Configuring
+sidebar_position: 30
+---
+
+import { Alert } from "@site/src/components/Alert";
+import { CodeExample } from "@site/src/components/CodeExample";
+
+Pre-requisites:
+
+- [Installation](./installing.mdx) is completed on a host machine and results in two executables: `stellar-horizon` and `stellar-core`. Per the installation instructions, `stellar-core` should be included in your host OS' `$PATH`, as the configurations below result in Horizon searching for a `stellar-core` to execute.
+- [Initialize Database](#initialize-horizon-database)
+
+You are now ready to setup configuration parameters to perform three important, distinct roles:
+
+- **serving requests** via a regular web-based HTTP API, and
+- **ingesting ledgers** from core nodes of a Stellar network to keep its world-view up to date, and
```

Comment:
@mollykarcher , @urvisavla , @Shaptic , wondering about a possible fourth role..a 'current state' role, a horizon instance that runs ingestion w/captive core, but stores no history, just the current state. provides cohort of api endpoints that come with that data only such as order book, offers, trades(trade-agg?), etc, not sure if enough app uses cases fall into this.

the role would equate to lighter-weight deployment, likely not external postgres/db, maybe get by with transient sqlite db spun up internally on disk/mem, this may have been evaluated already.

Reply:
>a horizon instance that runs ingestion w/captive core, but stores no history, just the current state.

Would this be same as running ingestion with no history retention? I just want to clarify my understanding of running ingestion with no history.